rust/rope/src/engine.rs

```diff
@@ -33,7 +33,7 @@ pub struct Engine {
 
 struct Revision {
     rev_id: usize,
-    from_union: Subset,
+    deletes_from_union: Subset,
```

Comment:
yes, this is a good name. probably main reason i didn't go there is i tend to favor terseness